marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/algorithms/schemaElements/matching-with-schema.xqy

```diff
@@ -0,0 +1,32 @@
+xquery version "1.0-ml";
+
+import module namespace algorithms = "http://marklogic.com/smart-mastering/algorithms" at
+  "/com.marklogic.smart-mastering/algorithms/double-metaphone.xqy",
+  "/com.marklogic.smart-mastering/algorithms/thesaurus.xqy",
+  "/com.marklogic.smart-mastering/algorithms/zip.xqy";
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+
+(: This is the minimum to test complex types defined in XML schema used with our match functions :)
```

Comment:
Does this mean this is doing matching based on an XML schema instead of a model?